modules/local/format_fasta_mirna.nf

```diff
@@ -0,0 +1,36 @@
+// Import generic module functions
+include { saveFiles; initOptions; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process FORMAT_FASTA_MIRNA {
+    label 'process_medium'
+
+    // publishDir "${params.outdir}",
```

Comment:
Why is this commented out? Shouldn't be necessary but you can set whether something is published or not in the modules.config.